src/LandIce/interface_with_mpas/Interface.cpp

```diff
@@ -412,6 +416,16 @@ void velocity_solver_extrude_3d_grid(int nLayers, int nGlobalTriangles,
     std::cout<<"\nWARNING: Using Dirichlet BCs options provided in Albany input file. In order to use those provided by MPAS, remove \"Dirichlet BCs\" sublist from Albany input file.\n"<<std::endl;
   }
 
+  if(paramList->sublist("Problem").isSublist("LandIce Field Norm") && paramList->sublist("Problem").sublist("LandIce Field Norm").isSublist("sliding_velocity_basalside"))
```

Comment:
@bartgol This is a bit ugly (you need to know how the field is named in the solver) but I think it is fine.

Reply:
The velocity is computed by Albany. That conditional is mainly for allowing to change the regularization (needed for schoof) in the input file. Eventually Mpas should provide the parameters